### PR TITLE
Update Safari Fullscreen API data

### PR DESCRIPTION
**Pull request by tomayac**
#### Summary

Update Fullscreen API data for Safari with version 17.4.

#### Test results and supporting details

https://developer.apple.com/documentation/safari-release-notes/safari-17_4-release-notes#Web-API

### CODE REVIEW

File: api/Document.json [hunk 1]
```diff
@@ -3103,7 +3103,10 @@
               {
                 "version_added": "16.4",
                 "partial_implementation": true,
-                "notes": "Only available on iPad, not on iPhone."
+                "notes": [
+                  "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline api/Document.json:3107]: To not say the same thing in two notes:

```suggestion
                  "Only available on iPad, not on iPhone.",
```

Can you make the same change throughout?

Comment (foolip) [reply api/Document.json:3107]: Oh, I'm sorry, I see now why you did this. I suggested it! That was a copy-paste-edit failure, sorry about that! I'll fix it.

File: api/Document.json [hunk 2]
```diff
@@ -3601,7 +3604,10 @@
...
```
Comment (foolip) [inline api/Document.json:3608]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Document.json [hunk 3]
```diff
@@ -3714,7 +3720,10 @@
...
```
Comment (foolip) [inline api/Document.json:3724]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Document.json [hunk 4]
```diff
@@ -3826,7 +3835,10 @@
...
```
Comment (foolip) [inline api/Document.json:3839]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Document.json [hunk 5]
```diff
@@ -3920,7 +3932,10 @@
...
```
Comment (foolip) [inline api/Document.json:3936]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Document.json [hunk 6]
```diff
@@ -4025,7 +4040,10 @@
...
```
Comment (foolip) [inline api/Document.json:4044]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Element.json [hunk 7]
```diff
@@ -4459,7 +4459,10 @@
               {
                 "version_added": "16.4",
                 "partial_implementation": true,
-                "notes": "Only available on iPad, not on iPhone."
+                "notes": [
+                  "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline api/Element.json:4463]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Element.json [hunk 8]
```diff
@@ -4571,7 +4574,10 @@
...
```
Comment (foolip) [inline api/Element.json:4578]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/Element.json [hunk 9]
```diff
@@ -7813,7 +7819,10 @@
               {
                 "version_added": "16.4",
                 "partial_implementation": true,
-                "notes": "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled."
+                "notes": [
+                  "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline api/Element.json:7823]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: api/HTMLIFrameElement.json [hunk 10]
```diff
@@ -156,7 +156,10 @@
             "safari_ios": {
               "version_added": "12",
               "partial_implementation": true,
-              "notes": "Only available on iPad, not on iPhone."
+              "notes": [
+                "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline api/HTMLIFrameElement.json:160]:
```suggestion
                "Only available on iPad, not on iPhone.",
```

File: api/ShadowRoot.json [hunk 11]
```diff
@@ -289,7 +289,10 @@
             "safari_ios": {
               "version_added": "16.4",
               "partial_implementation": true,
-              "notes": "Only available on iPad, not on iPhone."
+              "notes": [
+                "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline api/ShadowRoot.json:293]:
```suggestion
                "Only available on iPad, not on iPhone.",
```

File: css/selectors/fullscreen.json [hunk 12]
```diff
@@ -53,7 +53,10 @@
               {
                 "version_added": "16.4",
                 "partial_implementation": true,
-                "notes": "Only available on iPad, not on iPhone."
+                "notes": [
+                  "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline css/selectors/fullscreen.json:57]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

File: html/elements/iframe.json [hunk 13]
```diff
@@ -161,7 +161,10 @@
               "safari_ios": {
                 "version_added": "12",
                 "partial_implementation": true,
-                "notes": "Only available on iPad, not on iPhone."
+                "notes": [
+                  "Only available on iPad, not on iPhone. Shows an overlay button which can not be disabled.",
```
Comment (foolip) [inline html/elements/iframe.json:165]:
```suggestion
                  "Only available on iPad, not on iPhone.",
```

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
Thanks @tomayac! If you'd like to send a separate PR for the 17.4 changes we can get that reviewed and then merge around the time it's released.

**Comment by tomayac** [comment]
Perfect. Let's wait for the final release just to be on the safe side.

**Pull request merged**